exercises/practice/mazy-mice/.docs/instructions.md

```diff
@@ -0,0 +1,53 @@
+## Introduction
+
+Meet Mickey and Minerva, two clever mice who love to navigate their way through a maze to find cheese. 
+They enjoy a good challenge, but with only their tiny mouse brains, they prefer if there's only one correct path to the cheese.
+
```

Comment:
```suggestion
```

This should be (and is) part of the `introduction.md` file. We will automatically concatenate the files.

Reply:
done